### DISCUSSION TITLE
That a discussion is answered should always be visible and as visible from the bottom of the top of the discussion topic as in a mobile browser

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#51115] [Mobile] [answered]
GitHub Mobile beta

Visit a discussion that's been answered:
- https://github.com/community/community/discussions/15823

In mobile browsers, you can easily see the marked answer:

![Screenshot_20230327-091110.png](https://user-images.githubusercontent.com/2119212/227948309-cb54732b-63bf-4649-8e91-e3eca0104447.png)

In GitHub Mobile, unless you're at just the right place, you won't:
![Screenshot_20230327-091138.png](https://user-images.githubusercontent.com/2119212/227948325-b7322ac9-cea0-4139-8e6b-91b3dc436f92.png)

Asks (for after the current outage is fixed, although note that I intend to unmark that answer when the outage is resolved):

1. The top bar (to the left of the share icon) should gain a ✅
2. Clicking that ✅ should jump you to the marked answer
3. As with the mobile browser view, the marked answer should be previewed pinned just below the discussion topic, instead of only buried in the sea of answers

**Comment by stevepopovich** [comment, marked as answer]
This is coming in future releases, keep your eyes out for future updates!
